### PR TITLE
New Services MediaLive and MediaPackage

### PR DESCRIPTION
**Pull request by alistaim**
*Description of changes:*
Adding medialive and mediapackage models with minimal code examples for each.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/sdk/examples/medialive-helloworld/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#[tokio::main]
+async fn main() -> Result<(), medialive::Error> {
+    let client = medialive::Client::from_env();
+    let input_list = client.list_inputs().send().await?;
+
+    if let Some(inputs) = input_list.inputs {
```
Comment (rcoh) [inline aws/sdk/examples/medialive-helloworld/src/main.rs:6]: the pattern we use a lot for this is `input_list.unwrap_or_default()`

**Comment by rcoh** [comment]
Looks good—we're trying to migrate to one-crate-per-service with multiple binaries to make it a little more manageable when we have 260 services. Let me know if you need any guidance on making that refactoring. Otherwise LGTM!

File: aws/sdk/examples/medialive-helloworld/src/main.rs (hunk 1 above)
Comment (alistaim) [reply aws/sdk/examples/medialive-helloworld/src/main.rs:6]: Like this ?

`input_list.inputs.unwrap_or_default().iter().for_each(f)...`

File: aws/sdk/examples/medialive-helloworld/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        inputs.iter().for_each(|i| {
+            let input_arn = i.arn.to_owned().unwrap_or_default();
+            let input_name = i.name.to_owned().unwrap_or_default();
+
+            println!("Input Name : {}, Input ARN : {}", input_name, input_arn);
+        })
+    }
```
Comment (rcoh) [inline aws/sdk/examples/medialive-helloworld/src/main.rs:13]:
```suggestion
    for input in input_list.inputs.unwrap_or_default() {
          let input_arn = input.arn.as_deref().unwrap_or_default();
          let input_name = input.name.as_deref().unwrap_or_default();
          println!("Input Name : {}, Input ARN : {}", input_name, input_arn);
    }
```

Comment (rcoh) [reply aws/sdk/examples/medialive-helloworld/src/main.rs:13]: same pattern can be used in the other example

File: aws/sdk/examples/medialive-helloworld/src/main.rs (hunk 1 above)
Comment (rcoh) [reply aws/sdk/examples/medialive-helloworld/src/main.rs:6]: added github suggestion

File: aws/sdk/examples/medialive-helloworld/src/main.rs (hunk 2 above)
Comment (rcoh) [reply aws/sdk/examples/medialive-helloworld/src/main.rs:13]: `as_deref()` goes from `Option<T>` to `Option<&T>` when allows you do unwrap without taking ownership

Comment (alistaim) [reply aws/sdk/examples/medialive-helloworld/src/main.rs:13]: Awesome, let me make the changes.

**Review by rcoh — APPROVED** [review]

**Comment by rcoh** [comment]
Thanks!

**Pull request merged**